src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -662,19 +662,37 @@ public function setAutowiringTypes(array $types)
      */
     public function isAutowired()
     {
+        return !empty($this->autowired);
+    }
+
+    /**
+     * Gets autowired methods.
+     *
+     * @return string[]
+     */
+    public function getAutowired()
+    {
         return $this->autowired;
     }
 
     /**
      * Sets autowired.
      *
-     * @param bool $autowired
+     * Allowed values:
+     *   - true: constructor autowiring
+     *   - array('__construct', 'set*', 'setBar'): autowire whitelisted methods only
+     *
+     * @param string[]|bool $autowired
      *
      * @return Definition The current instance
      */
     public function setAutowired($autowired)
```

Comment:
@fabpot should it not support a boolean value? I like at least `autowire: true` for file loaders (opposed to `autowire: ['__construct']`).